lib/logging/addon/components/logging/form-test/template.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="over-hr mt-40">
+  <span>
+    {{#if testing}}
+      <button
+        class="btn bg-info btn-disabled text-white"
+        {{action "test"}}
```

Comment:
If you refactor this to use closure actions then you don't need the action in the component. It will simply call the passed action. 

The same applies for the rest of the file.